students/ssankura/lesson03/locke.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+'''
+Lesson03 - Activity 01 - Locke Implementation
+Author: Sireesha Sankuratripati
+'''
+class Locke(object):
+	def __init__(self, capacity_boats):
+		self.capacity_boats = capacity_boats
+
+	def __enter__(self):
+		print ("********* Entering the Locke  *********")
+		self.printUserMessage()
+		return self
+
+	def __exit__(self, type, value, tb):
+		print ("********* Exiting the Locke  *********")
+		self.printUserMessage()
+
+	def move_boats_through(self, num_boats):
+		if num_boats > self.capacity_boats:
+			raise ValueError("Capacity Error with number of boats -{}. Maximum number of Boats allowed is {}. ".format(num_boats, self.capacity_boats))
+		print ("*********  Moved {} boats through the Locke successfully  *********".format(num_boats))
+
+	def printUserMessage(self):
```

Comment:
might have been better to just define in the __init__ a `self.message` var which you could use in your enter and exit